### ISSUE TITLE
`ssh-add` existence?

### ISSUE THREAD

**Issue by treyharris** [blinksh/blink#1213]
### Checklist

- [x] ~~Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)~~ **Using TestFlight `Blink: v13.90.3.266. Mar  5 2021`**
- [X] Read the [docs](http://blink.sh/docs/)
- [X] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Describe the bug

If you type `ssh-` and <key>⇥</key>, one of the choices as you cycle through is `ssh-add`. Browsing the repo, it appears to be partially implemented, but  

```console
blink> ssh-add
ssh-add: command not found
blink> ssh-add -l
ssh-add: command not found
blink> ssh-add -L
ssh-add: command not found
```

I don’t see an issue tracking it as a FR. I’m submitting this as a bug report because autocomplete shouldn’t offer it up if it’s not a working command.

**Comment by carloscabanero** [comment]
Thanks! We have a new agent coming up. So instead of fixing this we will just provide the new command. It will be a few days only.

**Comment by carloscabanero** [comment]
This is now available on 13.90.3.273!
